05-descriptive-analysis.Rmd

```diff
@@ -132,7 +134,7 @@ Measures of central tendency find the central (or average) responses. These incl
 
 * Examples: Average 2000 Academic Performance Index in California, the median house price in Canada
 
-The `survey_mean()` and `survey_median()` functions from {srvyr} calculate mean and median using survey data, respectively. These functions must be called from within `summarize()`.
+The `survey_mean()` and `survey_median()` functions from {srvyr} calculate mean and median using survey data, respectively. These functions must be called from within `summarize()`.  
```

Comment:
All except survey_count and survey_tally must be within summarize. Let's make that point clearly and before getting into these groupings